tasks/sync-granule/tests/sync_granule_test.js

```diff
@@ -273,6 +273,37 @@ test('download granule with checksum in file from an HTTP endpoint', async (t) =
   }
 });
 
+test('download granule with bad checksum in file from HTTP endpoint throws', async(t) => {
+  const granuleChecksumValue = 8675309;
+
+  // Give it a bogus checksumValue to prompt a failure in validateChecksumFile
+  t.context.event.input.granules[0].files[0].checksumValue = granuleChecksumValue;
+  t.context.event.config.provider = {
+    id: 'MODAPS',
+    protocol: 'http',
+    host: 'http://localhost:3030'
+  };
+
+  validateConfig(t, t.context.event.config);
+  validateInput(t, t.context.event.input);
+
+  try {
+    // Stage the files to be downloaded
+    const granuleFilename = t.context.event.input.granules[0].files[0].name;
+    const granuleChecksumType = t.context.event.input.granules[0].files[0].checksumType;
+    const errorMessage = `Invalid checksum for ${granuleFilename} with type ${granuleChecksumType} and value ${granuleChecksumValue}`;
+    console.log(errorMessage);
+
+    await t.throws(syncGranule(t.context.event), errorMessage);
+  }
+  catch (e) {
+    if (e instanceof errors.RemoteResourceError) {
```

Comment:
What is the scenario you get this in? I think @yjpa7145 had some ideas on this before, so check with him. We want to avoid skipping tests.

Reply:
I'm in favor of removing these checks.  If there's a problem with the local test servers then the test should fail.

Reply:
👍

Reply:
That sounds reasonable to me. 👍